charts/kube-otel-stack/values.yaml

```diff
@@ -244,9 +251,190 @@ metricsCollector:
       pipelines:
         metrics:
           receivers: [prometheus, otlp]
-          processors: [memory_limiter, resource, resourcedetection/env, k8sattributes, metricstransform/k8sservicename, batch]
+          processors:
+            [
+              memory_limiter,
+              resource,
+              resourcedetection/env,
+              k8sattributes,
+              metricstransform/k8sservicename,
+              batch,
+            ]
           exporters: [otlp]
 
+logsCollector:
+  name: logs
+  clusterName: ""
+  image: otel/opentelemetry-collector-contrib:0.76.3
+  enabled: false
+  mode: daemonset
+  resources:
+    limits:
+      cpu: 250m
+      memory: 250Mi
+    requests:
+      cpu: 250m
+      memory: 250Mi
+  env:
+    - name: LS_TOKEN
+      valueFrom:
+        secretKeyRef:
+          key: LS_TOKEN
+          name: otel-collector-secret
+  volumeMounts:
+    - mountPath: /var/log
+      name: varlog
+      readOnly: true
+    - mountPath: /var/lib/docker/containers
+      name: varlibdockercontainers
+      readOnly: true
+  terminationGracePeriodSeconds: 30
+  volumes:
+    - name: varlog
+      hostPath:
+        path: /var/log
+    - name: varlibdockercontainers
+      hostPath:
+        path: /var/lib/docker/containers
+  config:
+    receivers:
+      filelog:
+        include:
+          - /var/log/pods/*/*/*.log
+        exclude:
+          # Exclude logs from all containers named otel-collector
+          - /var/log/pods/*/otel-collector/*.log
+        start_at: beginning
+        include_file_path: true
+        include_file_name: false
+        operators:
```

Comment:
How did you generate this config?

Reply:
Apologies for the delay! The logging filelog config was taken from the contrib examples repo [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/examples/kubernetes/otel-collector-config.yml). Let me know if I should link back to it.

Reply:
I added a comment with a link back to the examples in the contrib repo for reference.